src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -172,8 +172,9 @@ public XamlCodeGeneration(GeneratorExecutionContext context)
 			var applicationDefinitionItems = GetWinUIItems("ApplicationDefinition").Concat(GetWinUIItems("UnoApplicationDefinition"));
 
 			var xamlItems = pageItems
+				.Except(applicationDefinitionItems, MSBuildItem.IdentityComparer)
 				.Concat(applicationDefinitionItems)
```

Comment:
Except(applicationDefinitionItems).Concat(applicationDefinitionItems) is quite confusing 😕

Reply:
It's the order, I want to keep the original itemgroup, name favoring the `ApplicationDefinition` over the `Page.